apps/wear/smartdrive/app/pages/modals/updates/updates-view-model.ts

```diff
@@ -531,7 +531,7 @@ export class UpdatesViewModel extends Observable {
       const fwName = f['_filename'];
       const current = this.currentVersions[fwName];
       const currentVersion = current && current.version;
-      const isMax = v === maxes[fwName];
+      const isMax = v === allMaxes[fwName];
```

Comment:
```suggestion
      const isMax = v === reducedMaxes[fwName];
```
changing name to be consistent with other renames of similar code.